db/migrate/20230501161349_create_income_limits_tables.rb

```diff
@@ -0,0 +1,90 @@
+class CreateIncomeLimitsTables < ActiveRecord::Migration[6.1]
+  def change
+    create_table :std_counties do |t|
+      t.string :name, null: false
+      t.integer :countynumber, null: false
+      t.string :description, null: false
+      t.integer :state_id, null: false
+      t.integer :version, null: false
+      t.datetime :created, null: false
+      t.datetime :updated
+      t.string :createdby
+      t.string :updatedby
+    end
+    create_table :gmt_thresholds do |t|
```

Comment:
**Nitpick:** There should be an empty line above each `create_table` method instance.